parmed/openmm/parameters.py

```diff
@@ -205,12 +208,10 @@ def write(self, dest, provenance=None, write_unused=True, separate_ljforce=False
                 warnings.warn('Some residue templates are using unavailable '
                               'AtomTypes', ParameterWarning)
 
-        # DEBUG
-        print('Determining valid patch combinations...')
         valid_patch_combinations = self._determine_valid_patch_combinations(skip_residues)
+        LOGGER.debug('Valid patch combinations:')
         for patch_name in self.patches:
-            print('%8s : %s' % (patch_name, valid_patch_combinations[patch_name]))
-        print('')
+            LOGGER.debug('%8s : %s' % (patch_name, str(valid_patch_combinations[patch_name])))
```

Comment:
logging functions automatically do %-substitution (and this is caught by pylint, which I'd eventually like to use as a linter instead of pyflakes).  So change this line to

`LOGGER.debug('%8s : %s', patch_name, valid_patch_combinations[patch_name])`

`%s` also automatically calls `__str__`, so `str()` is unnecessary in that case.

Reply:
Done